### DISCUSSION TITLE
Calculation of thermal conductivity for two-dimensional material

### DISCUSSION THREAD

**Discussion by chejunwei2** [ttadano/alamode#54] [Q&A]
Dear developer,
I used ALAMODE to calculate the thermal conductivity of  two-dimensional material. I successfully got the xxx.kl file. So I wonder if the thermal conductivity in the xxx. kl file is the thermal conductivity of two-dimensional material, or if I need to post-process the calculated thermal conductivity according to the thickness of material. Thanks very much.

**Comment by ttadano** [comment]
Hi,

To obtain the thermal conductivity of 2D systems, you need to multiply the data in *.kl by (c/d) where c is the unit-cell length along the c-axis (perpendicular direction to the 2D sheet) and d is the thickness of the film. Please also see #32 for further details.

**Reply by chejunwei2** [reply]
Thanks very much for your reply.
Best regards.
